src/index.html

```diff
@@ -7,6 +7,100 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--30">
+      <div class="calendar__day calendar__day--sun">
+        <div class="calendar__day-1"></div>
+      </div>
+      <div class="calendar__day">
+        <div class="calendar__day-2"></div>
+      </div>
```

Comment:
please check again a task

Create a markup for the calendar block with 31 days inside
DON'T add numbers in HTML (you will do it using CSS)

```suggestion
      <div class="calendar__day"></div>
```